xarray/tests/__init__.py

```diff
@@ -112,6 +113,11 @@ def _importorskip(
     not has_h5netcdf_ros3[0], reason="requires h5netcdf 1.3.0"
 )
 
+has_netCDF4_1_6_2_or_above = _importorskip("netCDF4", "1.6.2")
```

Comment:
```suggestion
has_netCDF4_1_6_2_or_above,  requires_netCDF4_1_6_2_or_above =  _importorskip("netCDF4", "1.6.2")
```
And delete the line below.